.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.0","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/jackgilman/Library/Application Support/JetBrains/IntelliJIdea2020.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/Users/jackgilman/Library/Application%20Support/JetBrains/IntelliJIdea2020.1/plugins/Scala/launcher/sbt-launch.jar"]}
```

Comment:
This file needs to be removed (and added to .gitIgnore to prevent it being pushed up again)